src/ProjectService.Data/ProjectUserRepository.cs

```diff
@@ -78,10 +78,10 @@ public async Task<List<DbProjectUser>> GetAsync(List<Guid> usersIds)
       return await _provider.ProjectsUsers.Where(u => usersIds.Contains(u.UserId) && u.IsActive).ToListAsync();
     }
 
-    public async Task<List<Guid>> GetExistAsync(Guid projectId, IEnumerable<Guid> usersIds)
+    public async Task<List<DbProjectUser>> GetExistingUsersAsync(Guid projectId, IEnumerable<Guid> usersIds)
     {
       return await _provider.ProjectsUsers
-        .Where(pu => pu.IsActive && pu.ProjectId == projectId && usersIds.Contains(pu.UserId)).Select(pu => pu.UserId).ToListAsync();
+        .Where(pu => pu.ProjectId == projectId && usersIds.Contains(pu.UserId)).ToListAsync();
```

Comment:
Should you check if user isActive?

Reply:
no - this method is to check if user is in project now (even with isActive - false) in createProjectUser command